lib/shopify-cli/commands/create/project.rb

```diff
@@ -5,47 +5,106 @@ module Commands
     class Create
       class Project < ShopifyCli::SubCommand
         options do |parser, flags|
-          parser.on('--type=TYPE') do |t|
-            flags[:type] = t.downcase.to_sym
-          end
+          parser.on('--type=TYPE') { |t| flags[:type] = t.downcase.to_sym }
+          parser.on('--app_url=APPURL') { |url| flags[:app_url] = url }
+          parser.on('--app_url=APPURL') { |t| title[:title] = t }
```

Comment:
I love this, but I think you want app_url and app_title here?

Reply:
Yup! this PR is a mess and not ready for review yet!